modules/accredited_representative_portal/app/models/accredited_representative_portal/verified_representative.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  # Represents a verified representative within the Accredited Representative Portal.
+  # This class is responsible for managing the data associated with individuals who have
+  # been verified as representatives by the ARF Team. The model includes validations to ensure the presence and
+  # uniqueness of identifiers such as the OGC registration number and email.
+  #
+  # Currently, this model is populated manually by engineers as users are accepted into the pilot program.
+  # There is potential for a UI to be developed in the future that would facilitate administrative tasks
+  # related to managing verified representatives.
+  #
+  # A more automated process may be possible once OGC and MPI data facilitate such a process.
+  #
+  # == Associations
+  # This model may eventually be associated with AccreditedIndividuals to pull POA codes,
+  # if they exist, based on the OGC registration number. It currently does so via a helper method.
+  class VerifiedRepresentative < ApplicationRecord
+    validates :ogc_registration_number, presence: true, uniqueness: { case_sensitive: false }
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :email, presence: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
```

Comment:
FYI, @raudabaugh found that a small number of reps currently share email addresses in the OGC data set. We're thinking we should require reps to have unique email addresses to log into ARP, so we should code in an error case for this that will result in a message being shown to the user

Reply:
Done! See commit:
a66de491aa

Description:
- Implement error handling for scenarios where a VerifiedRepresentative has a non-unique or non-existent email address in the associated AccreditedIndividual records.
- For the pilot, log errors without raising exceptions to ensure a smooth user experience.
- In the long term, consider raising errors and displaying appropriate messages to users when such inconsistencies are encountered.

Discussion:
- @raudabaugh discovered that a small number of representatives currently share email addresses in the OGC data set.
- To maintain data integrity and simplify the login process for ARP, it is recommended to require representatives to have unique email addresses.
- Allowing non-unique email addresses introduces complexity in two directions:
  1. Using user-entered OGC registration numbers for authentication, which are unique but public, essentially trusting users to provide accurate information.
  2. Using first and last names for authentication, which are not unique and can change (e.g., due to marriage), potentially leading to loss of access to ARP.
- For the pilot, the current approach of logging errors without raising exceptions is sufficient.
- In the long term, consider implementing a probabilistic approach (similar to MPI) for identity matching based on names, rather than a deterministic one, to avoid painful access scenarios for users.

CC: @raudabaugh @amprokop